src/segment.rs

```diff
@@ -376,10 +403,13 @@ impl Segment {
         let end = self.size();
 
         match start.cmp(&end) {
-            Ordering::Equal => Ok(()), // nothing to flush
+            Ordering::Equal => {
+                trace!("{:?}: nothing to flush", self);
+                Ok(())
+            } // nothing to flush
```

Comment:
I propose deleting the comment because `nothing to flush` logic is already mentioned in log message